blues/example.py

```diff
@@ -11,12 +11,11 @@
 """
 
 from __future__ import print_function
-from blues import ncmc
-from blues import utils
+from blues.moves import RandomLigandRotationMove, MoveEngine
```

Comment:
My reasoning for doing it this way was so if someone copies the `example.py` script it should work for them no matter where it's located. I think being more explicit about things is helpful in example scripts too.